lib/adapter.js

```diff
@@ -6961,7 +6950,7 @@ function Adapter(options) {
 
                         // read all aliases
                         this.getForeignObjects(pattern, null, null, options, (err, objs) => {
```

Comment:
don't we have an Async version of this method we could use/await here?
Lets start getting rid of the callback hell :)